book_maker/translator/chatgptapi_translator.py

```diff
@@ -37,6 +40,7 @@
     "gpt-4o-mini-2024-07-18",
 ]
 CONTEXT_PARAGRAPH_LIMIT = 3
+BATCH_CONTEXT_UPDATE_INTERVAL = 50
```

Comment:
let's make a config.py file for these?

Reply:
@yihong0618 
I created config.py and modified the code to use it. Since this is my first time creating config.py, I'm not sure if I'm using it correctly. Could you please review it?

[fd92f7a](https://github.com/yihong0618/bilingual_book_maker/pull/423/commits/fd92f7ae56eeee821907abfe4521315848599431)